### PR TITLE
Turn :company-prefix-length property into a value

### PR DESCRIPTION
**Pull request by fbergroth**
As suggested in #450.

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 ## Next
 
+* `company-capf` property `:company-prefix-length` is now a value.
```
Comment (dgutov) [inline NEWS.md]: Please edit the original entry instead.

Comment (fbergroth) [inline NEWS.md]: Updated.

**Comment by dgutov** [comment]
:+1:

I wasn't waiting for you to do this, but thanks for taking the initiative!

**Comment by fbergroth** [comment]
My pleasure :smile:

**Pull request merged**